test/api.test.js

```diff
@@ -1,97 +1,100 @@
-describe('API player tests', () => {
-  beforeEach(async () => {
-    await page.setViewport({width: 1280, height: 1800});
-    await page.goto('http://localhost:3000/api.html', {waitUntil: 'load'});
-    await page.evaluate(() => {
-      Object.defineProperty(HTMLMediaElement.prototype, 'playing', {
-        get: function() {
-          return !!(this.currentTime > 0 && !this.paused && !this.ended &&
+const urls = ['http://localhost:3000/api.html', 'http://localhost:3000/api.html?testUrlWithGet=true'];
```

Comment:
@maoznir , come to think about it, why not having a param of testUrlWith=get/head?